.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+### Motivation
```

Comment:
I don't like to include markdown as not all tools support it and it looks ugly if they not do. For example I often use `git log` on the cmdline etc.